index.js

```diff
@@ -422,6 +422,8 @@ class Raspi extends EventEmitter {
     const pinInstance = this[getPinInstance](this.normalize(pin));
     if (pinInstance.mode === INPUT_MODE && value === HIGH) {
       this[pinMode]({ pin, mode: INPUT_MODE, pullResistor: PULL_UP });
+    } else if (pinInstance.mode === INPUT_MODE && value === LOW) {
```

Comment:
Thanks! We can leave this PR open for now, but I want to wait to land it until the discussion for Johnny-Five is resolved.

Just an FYI, this module is specifically designed to be a shim between Johnny-Five and [Raspi.js](https://github.com/nebrius/raspi). So from a design perspective, it doesn't make sense to add anything here that Johnny-Five specifically isn't requesting.

If you want to use raspi-io w/o Johnny-Five, I would recommend using Raspi.js as it's a little easier to use IMO.